### DISCUSSION TITLE
question on keybindings

### DISCUSSION THREAD

**Discussion by germanier** [i3/i3#4545] [Q&A]
hey all!
kinda new to dot files
was trying to make the script ./rofi-beats in home/rxx/Rofi-Beats run on a key combination, can't figure out the syntax:

bindsym $mod+Shift+p exec home/rxx/Rofi-Beats ./rofi-beats

what's the correct syntax? thanks!

**Comment by Airblader** [comment]
```
bindsym $mod+Shift+p exec home/rxx/Rofi-Beats/rofi-beats
```

This assumes that the script is actually executable, of course.

**Reply by germanier** [reply]
if i cd/Rofi-Beats and do ./rofi-beats it does work, but the above doesn't

the script in question is from https://github.com/Carbon-Bl4ck/Rofi-Beats

did chmod +x on rofi-beats just in case, and now it doesn't show a red flag, but nothing happens...

**Reply by Airblader** [reply]
You could try redirecting stdout+stderr to some file and seeing if anything pops up. It might be a path issue. i3 simply forwards the part after `exec` to a shell.
